Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -1521,6 +1525,12 @@
                             "$ref": "#/components/schemas/DeclarationToken"
                         }
                     },
+                    "images": {
```

Comment:
Will we add some sort of `pageIcon` to the page `metadata`, so we know to replace the icon in the DocHero?

Reply:
Yes! That should already be below: https://github.com/apple/swift-docc/pull/367/files#diff-fc7c8431152d3d494899ec4b551f1a5d1249ed44c853e05da1cc26374d9d4c54R2036